lib/messages/chainlock.js

```diff
@@ -0,0 +1,276 @@
+var _ = require('lodash');
```

Comment:
Why does it in the `messages` directory? Almost every class in the lib represents a p2p message. Also, the lib is domain-driven, i.e. it's scoped not by the type of object, but rather by its domain. For example the `script` directory has the script interpreter class, as well as script parser and script model

Reply:
Please see this : https://github.com/dashevo/dashcore-lib/pull/176#discussion_r499530355

Reply:
We had a conversation with Ivan regarding this comment, and the current agreement is to just have ChainLock class